project.clj

```diff
@@ -52,6 +52,7 @@
                  ;; Data types, encoding, etc.
                  [byte-streams "0.2.0"]
                  [clj-time "0.11.0"]
+                 [commons-codec "1.9"]
```

Comment:
Do you mean... 

https://github.com/clojure/data.codec

? ...can do, looks like it fits the bill even though it only implements base64. If there is another call built into Clojure core that's escaping me, let me know. I know Java 8 finally ads base64 encoding but think we are targeting Java 7. 

;j-

Sent from my Turing Machine

> On Feb 9, 2016, at 18:13, Duncan McGreggor <notifications@github.com> wrote:
> 
> In project.clj:
> 
> > @@ -52,6 +52,7 @@
> >                   ;; Data types, encoding, etc.
> >                   [byte-streams "0.2.0"]
> >                   [clj-time "0.11.0"]
> > +                 [commons-codec "1.9"]
> Did you try using the native Clojure library? (https://github.com/clojure/data.codec)
> 
> That's what I've used in the past ...
> 
> —
> Reply to this email directly or view it on GitHub.
>